input_definition.go

```diff
@@ -111,14 +115,25 @@ func (i *InputDefinition) LoadDefinition(pb *internal.InputDefinition) error {
 			})
 		}
 
+		if field.PrimaryKey {
```

Comment:
Similar to my comment in `handler.go`, this renaming of the field is not correct.